.github/workflows/test.yaml

```diff
@@ -213,3 +213,29 @@ jobs:
           PROPTEST_CASES: "10"
           MIRIFLAGS: "-Zmiri-disable-isolation"
         run: cargo miri test --all-features
+
+  thread_sanitizer:
+    name: Thread Sanitizer checks
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          target/
+        key: ${{ github.event.repository.name }}-${{ runner.os }}-cargo-thread_sanitizer-v2
+    - name: Install latest nightly
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: nightly
+          override: true
+          components: rust-src
+    - name: Run thread sanitizer
+      run: |
+        export RUSTDOCFLAGS='-Zsanitizer=thread'
+        export RUSTFLAGS='-Zsanitizer=thread'
```

Comment:
Please use the `env` section for environment variables instead of shell exports.

Reply:
Fixed.